internal/nix/search.go

```diff
@@ -34,7 +37,7 @@ func Search(url string) (map[string]*Info, error) {
 		// TODO implement runx search
 		return map[string]*Info{}, nil
 	}
-	return searchSystem(url, "")
+	return searchSystemUsingCache(url, "")
```

Comment:
I think this is not the best. 

Instead, I'll do:
- revert this callsite to `searchSystem`. 
- rename `searchSystemUsingCache` to `SearchNixpkgAttribute`
- In [normalizePackageAttributePath()](https://github.com/jetpack-io/devbox/blob/b2a883596325a29b207756d325df2afbfe7b8cf8/internal/devpkg/package.go#L289-L303), I'll have `DevboxPackages` call `SearchNixpkgsAttribute`, while non-Devbox Packags continue calling `nix.Search`.

Reply:
done